### ISSUE TITLE
Better cache busting

### ISSUE THREAD

**Issue by ethteck** [decompme/decomp.me#1109]
When we update certain things like the recent #1103 , it breaks the site because the browser cache doesn't play well with the new state of things

**Comment by bates64** [comment]
vercel deploys would fix this

**Comment by ethteck** [comment]
I'm sure there's an elegant solution that wouldn't require using vercel. I would rather we go for a simple fix for this issue

**Comment by bates64** [comment]
yeah, i just maintain that vercel would give us lots of other nice benefits too

**Comment by bates64** [comment]
as vercel want me to think obviously so they can make their investors happy but its free so

**Comment by ethteck** [comment]
I think I've found one of the _issues._ /api/compiler is returning a 304. Once I manually loaded this endpoint, I stopped getting errors. adding the backend tag too

**Comment by ethteck** [comment]
I think `last_modified_func`s should add in when the backend launched: https://github.com/decompme/decomp.me/blob/main/backend/coreapp/views/compiler.py#L40
